modules/dynamic-plug-in-sdk-extensions.adoc

```diff
@@ -22,9 +22,10 @@
 contributed actions to a particular area of the application. Examples include `topology` and `helm`.
 
 |`filter` |`CodeRef<(scope: any, action: Action) => boolean>` |no |A
-function that will filter actions based on some conditions. `scope`: The
-scope in which actions should be provided for. Hook might be required
-if you want to remove the `ModifyCount` action from a deployment with HPA.
+function that will filter actions based on some conditions. `scope`: The scope
```

Comment:
@ahardin-rh We should fix this in the openshift/console repo or the next time we update the SDK doc the change will be reverted.

cc @opayne1 @jhadvig